packages/components/datagrid/src/components/osds-datagrid/interfaces/attributes.ts

```diff
@@ -19,6 +19,8 @@ interface OdsDatagridAttribute {
   rows: OdsDatagridRow[] | string;
   /** The rows can be selectable */
   isSelectable?: boolean
+  /** Text when the datagrid was no rows */
+  placeholder?: string;
```

Comment:
maybe not the most comprehensive name as it can be confusing with the input ones, `noResultLabel`?